### ISSUE TITLE
warning for 'class param without default' is not catching all instances in a file

### ISSUE THREAD

**Issue by pccowboy** [rodjek/puppet-lint#157]
I have:

```
# Define jenkins::repo::svn
#
# This module sets up an SVN branch to store a Jenkins configuration
#
# Parameters:
# * SVN host URL path
# * project directory on the SVN host
# * branch
#
# Actions:
# * Creates a new SVN branch under the project
#
# Requires:
# * SVN must be installed
# * don't try and run 2 jenkins servers on one machine...
#
class jenkins::repo::svn (
  $host,
  $project,
  $branch
) {

}
```

And when I run puppet-lint via the command line, I am getting:

```
WARNING: parameterised class parameter without a default value on line 18
WARNING: parameterised class parameter without a default value on line 19
```

There should be a warning issued for line 20 as well, shouldn't there?

In addition, a feature request - can a comment flag be set, to allow 'per-file' disabling of certain checks?  It would help me get around errors in my CI that are not really errors, like this one.

While I understand what the recommended "fix" is for this type of construct, in this case there is no sane default for any of the params - the hostname of the svn server, the project, and the branch all need to be provided by the user of this class.

A way for me to indicate to puppet-lint within this file that I have knowingly violated one of the "shoulds" would increase the signal:noise ratio from puppet-lint, I believe.
